### PR TITLE
Implement GPURenderPassDescriptor.maxDrawCount test

### PR DESCRIPTION
**Pull request by takahirox**
Issue: #1613

Test plan: #1614 

<hr>

**Requirements for PR author:**

- [x] All missing test coverage is tracked with "TODO" or `.unimplemented()`.
- [x] New helpers are `/** documented */` and new helper files are found in `helper_index.txt`.
- [x] Test behaves as expected in a WebGPU implementation. (If not passing, explain above.)

**Requirements for [reviewer sign-off](https://github.com/gpuweb/cts/blob/main/docs/reviews.md):**

- [ ] Tests are properly located in the test tree.
- [ ] [Test descriptions](https://github.com/gpuweb/cts/blob/main/docs/intro/plans.md) allow a reader to "read only the test plans and evaluate coverage completeness", and accurately reflect the test code.
- [ ] Tests provide complete coverage (including validation control cases). **Missing coverage MUST be covered by TODOs.**
- [ ] Helpers and types promote readability and maintainability.

When landing this PR, be sure to make any necessary issue status updates.

### CODE REVIEW

**Comment by takahirox** [comment]
From https://github.com/gpuweb/cts/pull/1614#discussion_r920869510

@Kangz Do you think we should test single-draw-type cases?

If we consider "Check adverserialy", the current test can even pass, for example, if the implementation counts up by two for `draw()` but doens't count up for `drawIndexed()`.

File: package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "@types/offscreencanvas": "^2019.6.4",
     "@types/serve-index": "^1.9.1",
     "@typescript-eslint/parser": "^4.33.0",
-    "@webgpu/types": "0.1.20",
+    "@webgpu/types": "0.1.21",
```
Comment (takahirox) [inline package.json:43]: Note to reviewers: 0.1.21 is needed for `GPURenderPassDescriptor.maxDrawCount`

**Comment by takahirox** [comment]
Friendly ping @Kangz

**Review by Kangz — APPROVED** [review]
Thank you for the ping. I tend to mostly ignore CTS Github emails :X

Test LGTM, it is very clear, and comprehensive while being a single test.

**Pull request merged**